src/apps/inventory/views.py

```diff
@@ -96,3 +99,41 @@ def form_valid(self, form):
 
     def test_func(self):
         return self.request.user.user_type == "Seller"
+
+
+class ProductUpdateView(LoginRequiredMixin, UpdateView):
```

Comment:
maybe add UserPassesTestMixin i think it looks cooler, but thats my opinion

Reply:
great point, we need to test if our user is actual seller of product